blocksuite/affine/data-view/src/view-presets/table/pc-virtual/controller/drag.ts

```
@@ -0,0 +1,211 @@
+  //         this.host.props.view.rowMove(
+  //           row.rowId,
+  //           result.position,
+  //           fromGroup,
+  //           result.groupKey
+  //         );
+  //       }
+  //     },
+  //   });
+  // };
+
+  dropPreview = createDropPreview();
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)